### DISCUSSION TITLE
The download is not progressing

### DISCUSSION THREAD

**Discussion by Fizionomik** [AveYo/MediaCreationTool.bat#23] [Q&A]
Hi, I wanted to install windows 11 and did everything right. The download just doesn't go.
![image](https://user-images.githubusercontent.com/52714102/139493543-6a76c8fe-8bc1-4734-b8d8-a69bcd96c721.png)

**Comment by AveYo** [comment]
maybe download servers were overloaded for your region, or maybe you were blocked by the servers for too many requests (temp-ban)
